x-pack/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -38,13 +42,23 @@ function TransactionBreakdownGraph(props: Props) {
     [trackApmEvent]
   );
 
+  const isTimeseriesEmpty = isEmpty(timeseries);
+  const chartHeight = isTimeseriesEmpty ? undefined : unit * 12;
```

Comment:
> Why do we change the height based on the chart being empty or not?

When there is data a custom height needs to be set, to not stretch the chart.
<img width="1369" alt="Screenshot 2020-08-20 at 13 15 33" src="https://user-images.githubusercontent.com/55978943/90763681-64854600-e2e7-11ea-8884-1071e6e8ea02.png">

> What happens when you set undefined?

Undefined makes the chart to set the default height, which is `256px` since when there's no data the `TransactionBreakdownKpiList` is not be visible, I did it to show all charts aligned. Otherwise, it'd show like:
<img width="1354" alt="Screenshot 2020-08-20 at 13 24 14" src="https://user-images.githubusercontent.com/55978943/90764345-7adfd180-e2e8-11ea-8f2c-66a9cb19833a.png">

Reply:
Ah, I see. I feel it makes more sense to not collapse the KPI list then if it's empty. This was something we previously did to save space but I don't think that's necessary anymore.

Reply:
I'd have to set a fixed height on `TransactionBreakdownKpiList` in that case, is that what you mean?

Reply:
@cauemarcondes yeah